code/mathematical_algorithms/src/newman_conway/README.md

```diff
@@ -0,0 +1,13 @@
+# Newman - Conway Number  
```

Comment:
Please follow our documentation guide. (I can't find a link to it)

Reply:
Never mind, I found it: https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md. Please follow this guide for this document.